docs/modules/migrate/pages/data-migration-tool.adoc

```diff
@@ -97,93 +196,177 @@ Windows::
+
+. Navigate to the folder in which you extracted the DMT package
+. Open the _migration_config/data/imap_names.txt_ and/or the _migration_config/data/replicated_map_names.txt_ file in your favorite editor
+. Update the file content to match the names of your maps. To select multiple data structures using a single entry, you can use wildcards. For further information on using wildcards, see the xref:configuration:using-wildcards.adoc[Using Wildcards] topic.
++
+NOTE: If you have multiple data structures, use a new line for each map name.
+
+. Save the file
+
+=== Check the Target Cluster
+
+Ensure that the target cluster is running on one of the following:
+
+* Enterprise Edition version 5.3.2 or later 
+* {hazelcast-cloud}
+
+==== Update the Target Configuration
+
+You must update the following configuration:
+
+* The cluster
+* The connection
+* If required, SSL
+
+To update this configuration, complete the following steps:
+
+. Navigate to the folder in which you extracted the DMT package
+. Open the _migration_config/target/hazelcast-client.yaml_ file in your favorite editor
++
+NOTE: The _hazelcast-client.yaml_ file is a Hazelcast client configuration file, which can include any supported configuration.
++
+. Update the `cluster-name` field to match the name of your source cluster
+. Update the `network` section as follows:
++
+* For an on-premise target cluster, update the `cluster-members` field to match the addresses of the cluster members
+* For a cloud target cluster, including a {hazelcast-cloud} cluster, update the network information. For a public cloud cluster, refer to the documentation for the cloud provider for the required network details. For {hazelcast-cloud}, you must update the network section as follows:
++
 ----
-docker run -e HZ_NETWORK_PORT_PORT=5901 -p 127.0.0.1:5901:5901 
-           -e HZ_LICENSEKEY="<license>" <1>
-           -e HZ_CLUSTERNAME=target hazelcast/hazelcast-enterprise:{full-version}-slim <2>
+network:
+    hazelcast-cloud:
+      enabled: true
+      discovery-token: <token>
 ----
-<1> If you don't have a Hazelcast Enterprise license key, you can either use a Hazelcast Viridian https://docs.hazelcast.com/cloud/free-trial[trial cluster] or request a xref:getting-started:enterprise-overview.adoc#step-2-get-your-enterprise-license[trial license].
-<2> If you want to use the full distribution, remove the `-slim` postfix. See xref:getting-started:editions.adoc#full-and-slim-distributions[Distributions] to learn about the differences between full and slim distributions.
 
-The cluster tries to start on port `5901` if it's available. It chooses another port otherwise. You can check the port in the cluster logs shown on the terminal.
+. If required, add the `ssl` information. The format is as follows:
++
+----
+ssl:
+      enabled: true
+      properties:
+        keyStore: client.keystore
+        keyStorePassword: abc123
+        trustStore: client.truststore
+        trustStorePassword: abc123
```

Comment:
This syntax is incorrect, the ssl section is under hazelcast/network, so the sample should be:

```suggestion
hazelcast:
  network:
    ssl:
      enabled: true
      properties:
        keyStore: client.keystore
        keyStorePassword: abc123
        trustStore: client.truststore
        trustStorePassword: abc123
```

Reply:
Updated